### ISSUE TITLE
build error

### ISSUE THREAD

**Issue by elderbig** [BinChengZhao/delay-timer#30]
error when build version 0.10.0
```
cargo clean
cargo update
cargo build
```

error message
```
error[E0308]: mismatched types
   --> src/utils/parse.rs:308:9
    |
308 |         Ok(process_linked_list)
    |         ^^^^^^^^^^^^^^^^^^^^^^^ expected enum `error::CommandChildError`, found struct `anyhow::Error`
    |
    = note: expected enum `std::result::Result<_, error::CommandChildError>`
               found enum `std::result::Result<_, anyhow::Error>`

For more information about this error, try `rustc --explain E0308`.
error: could not compile `delay_timer` due to previous error

```

**Comment by BinChengZhao** [comment]
> error when build version 0.10.0
> 
> ```
> cargo clean
> cargo update
> cargo build
> ```
> 
> error message
> 
> ```
> error[E0308]: mismatched types
>    --> src/utils/parse.rs:308:9
>     |
> 308 |         Ok(process_linked_list)
>     |         ^^^^^^^^^^^^^^^^^^^^^^^ expected enum `error::CommandChildError`, found struct `anyhow::Error`
>     |
>     = note: expected enum `std::result::Result<_, error::CommandChildError>`
>                found enum `std::result::Result<_, anyhow::Error>`
> 
> For more information about this error, try `rustc --explain E0308`.
> error: could not compile `delay_timer` due to previous error
> ```

Friend, thanks for your report, I have fixed the bug and can use version:`0.10.1`.

Because I recently joined a new company, things are especially and busy, your remaining two problems I will find the opportunity to solve.

But any serious, blocking errors will be my first concern.

**Comment by elderbig** [comment]
issue prepared
